src/ic3/aux_types.hh

```diff
@@ -6,8 +6,14 @@ Author: Eugene Goldberg, eu.goldberg@gmail.com
 
 ******************************************************/
 
+#pragma once
```

Comment:
We don't usually use this in this code base. Why is it necessary here?

Reply:
None of those ic3 headers appear to use a header guard, so omit altogether if it compiles, I'd say.